src/db/models/Commit.ts

```diff
@@ -18,3 +18,9 @@ export async function createCommit(commit: Commit): Promise<Commit> {
   const commits = await db(tableName).returning("*").insert<Commit[]>([commit]);
   return commits[0];
 }
+
+export async function getCommits(afterTimestamp: Date): Promise<Commit[]> {
```

Comment:
The name should indicate the behavior more, since parameter names are not shown at call site. Maybe `getCommitsAfterDate`?